commands.json

```diff
@@ -1735,15 +1735,15 @@
     "step": 1
   },
   "xreadgroup": {
-    "arity": -3,
+    "arity": -1,
     "flags": [
       "write",
       "noscript",
       "movablekeys"
     ],
-    "keyStart": 1,
-    "keyStop": 1,
-    "step": 1
```

Comment:
Would you mind updating the whole `commands.json` instead of manually updating this entry? That can be done by running `npm run build` in combination with the newest Redis version.

Reply:
Done and done. This is my first contribution ever, thank you for your patience.